packages/provider/jest.config.js

```diff
@@ -22,7 +22,10 @@ module.exports = {
   // original implementations, between each test. It does not affect mocked
   // modules.
   restoreMocks: true,
-  testEnvironment: 'node',
+  testEnvironment: 'jsdom',
+  testEnvironmentOptions: {
+    customExportConditions: ['node', 'node-addons'],
```

Comment:
Can we remove this now?

Reply:
No, we need that regardless of environment AFAIK.